r55/tests/e2e.rs

```diff
@@ -0,0 +1,40 @@
+use alloy_sol_types::SolValue;
+use r55::{
+    compile_deploy,
+    exec::{deploy_contract, run_tx},
+    test_utils::add_balance_to_db,
+};
+use revm::{
+    primitives::{address, keccak256, Address, Bytes},
+    InMemoryDB,
+};
+
+#[test]
+fn erc20() {
+    const ERC20_PATH: &str = concat!(env!("CARGO_MANIFEST_DIR"), "/../erc20");
+    let rv_bytecode = compile_deploy(ERC20_PATH).unwrap();
+    let mut db = InMemoryDB::default();
+
+    let mut bytecode = vec![0xff];
+    bytecode.extend_from_slice(&rv_bytecode);
+
```

Comment:
we could extract a helper that does the compilation and adds the `0xff` prefix

Reply:
Agree. Implemented a new method `compile_with_prefix`.